nmclicktests/environment.js

```diff
@@ -193,107 +20,54 @@ const prependLines = (pre, text) => {
 // Environment provides
 class Environment {
   constructor(config) {
-    this.nm = Nightmare({ Promise: Bluebird, typeInterval: 500, show: false });
     this.config = config || {};
     this.config.rootPath = (typeof this.config.rootPath === 'string') ? this.config.rootPath : '';
-    this.config.serverTimeout = this.config.serverTimeout || 15000;
+    this.config.serverTimeout = this.config.serverTimeout || 35000;
     this.config.viewWidth = 2000;
```

Comment:
btw any idea why this viewport is so big and square ?

Maybe we can give it more sense and switch to a FullHD viewport which represents better the reality ?

Reply:
Not sure why this was done in https://github.com/FredrikNoren/ungit/commit/c7cdddca9083be7c03acdc37d64763f2b7688658in https://github.com/FredrikNoren/ungit/pull/908
But I will try out if it works in 1920 × 1080.

Reply:
Yes, they work in full HD!